Formula/arangodb.rb

```diff
@@ -16,14 +18,38 @@ class Arangodb < Formula
   depends_on :macos => :yosemite
   depends_on "openssl"
 
+  # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87665
+  fails_with :gcc do
+    build 730
```

Comment:
This should probably replace the block below

Reply:
I wanted to express that both gcc 7.3.0 and gcc 8.2.0 have this issues. Is there a better way to do it?